tests/compute/test_heterograph.py

```diff
@@ -1929,6 +1929,134 @@ def test_edges_order():
     assert F.array_equal(F.copy_to(dst, F.cpu()),
                          F.copy_to(F.tensor([1, 1, 2, 2, 1]), F.cpu()))
 
+@parametrize_dtype
+def test_reverse(index_dtype):
+    g = dgl.heterograph({
+        ('user', 'follows', 'user'): ([0, 1, 2, 4, 3 ,1, 3], [1, 2, 3, 2, 0, 0, 1]),
+    }, index_dtype=index_dtype)
+    gidx = g._graph
+    r_gidx = gidx.reverse()
```

Comment:
It's better not to test the APIs of graph index. They are not user-facing.

Reply:
This is for end2end test.
Otherwise, I can not test the overall path.